.github/workflows/build_nix.yml

```diff
@@ -0,0 +1,13 @@
+name: "Build legacy Nix package on Ubuntu"
```

Comment:
One last thing: what's the purpose of this workflow? I guess this is only to test that it successfully builds with nix, right?

Reply:
Yes!